classes/ConnectionsSource.php

```diff
@@ -51,26 +51,6 @@ class ConnectionsSourceCore extends ObjectModel
         ],
     ];
 
-    /**
-     * Adds current ConnectionsSource as a new Object to the database.
-     *
-     * @param bool $autoDate Automatically set `date_upd` and `date_add` columns
-     * @param bool $nullValues Whether we want to use NULL values instead of empty quotes values
-     *
-     * @return bool Indicates whether the ConnectionsSource has been successfully added
-     *
-     * @throws PrestaShopDatabaseException
-     * @throws PrestaShopException
-     */
-    public function add($autoDate = true, $nullValues = false)
-    {
-        if ($result = parent::add($autoDate, $nullValues)) {
-            Referrer::cacheNewSource($this->id);
-        }
-
-        return $result;
-    }
-
     public static function logHttpReferer(Cookie $cookie = null)
```

Comment:
I would naively think that if the Referrer feature is removed then this whole method becomes useless? Unless it's badly named and is used for other purposes?

But at first sight, I'm thinking: if don't handle Referrer anymore then we never need to log them no?

Reply:
It's badly named. It allows to log connection source (and before this PR, it logged referrer to this connection source).